tests/DFE.FindInformationAcademiesTrusts.CypressTests/cypress/pages/commonPage.ts

```diff
@@ -0,0 +1,43 @@
+class CommonPage {
+
+    elements = {
+        SuccessPopup: {
+            Section: () => cy.get('.govuk-notification-banner'),
+            Message: () => this.elements.SuccessPopup.Section().find('.govuk-notification-banner__content')
+        },
+
+        ErrorPopup: {
+            Section: () => cy.get('.govuk-error-summary'),
+            Message: () => this.elements.ErrorPopup.Section().find('[data-testid="error-summary"]')
+        },
+
+        Datasource: {
+            Section: () => cy.get('.govuk-details'),
+            LatestUpdatedBy: () => cy.get('[data-testid="data-source-fiatdb-trust-relationship-manager"] > :nth-child(3)')
```

Comment:
I think this need another name as its just the last updated by for the trust relationship manager. It also needs to be moved to the contacts page as it's not common to any other pages.

Reply:
Also do we need to add one to check the sfso name has updated too?

Reply:
Also not a fan of using nth child, I'd probably chain the get with a contains("Updated by:")

Reply:
Now done both of the above in latest round of fixes